proposals/0026-hlsl-long-vector-type.md

```diff
@@ -105,8 +129,9 @@ myBuffer.Store(elementIndex, val);
 
 #### Accessing elements of long vectors
 
-Long vectors support the existing vector subscript operators to return the scalar element values.
-They do not support swizzle operations as they are limited to only the first four elements.
+Long vectors support the existing vector subscript operators `[]` to access the scalar element values.
+They do not support any swizzle operations.
+Swizzle operations are limited to the first four elements and the accessors are named according to the graphics domain.
```

Comment:
These two sentences seem contradictory now.  I think the original version was clearer - swizzle isn't supported, because of reasons.  Is the intent to change this to say that they're supported now for just the first four elements?

Reply:
I don't think they are. The first says that long vectors don't support swizzle operations. The second describes what they are to try to explain why. I'm not sure the rationale needs to be here though. I can put it in alternatives considered.